src/grt/src/GlobalRouter.tcl

```diff
@@ -193,6 +193,7 @@ proc set_global_routing_random { args } {
 sta::define_cmd_args "global_route" {[-guide_file out_file] \
                                   [-congestion_iterations iterations] \
                                   [-congestion_report_file file_name] \
+                                  [-drc_report_iter_step] \
```

Comment:
this takes an argument eg [-drc_report_iter_step step]

Reply:
The argument was added